apps/portals/src/_Core.scss

```diff
@@ -403,12 +403,6 @@ button.dropdown-toggle.nav-button:hover {
 }
 // end header
 
-#explore-portal {
```

Comment:
I went through a few of our scss files and deleted styles that mapped to classes or IDs that didn't appear in a full project search, where I was reasonably confident they didn't come from a third party lib.